package.json

```diff
@@ -24,7 +24,9 @@
     "test:lint": "eslint \"**/*.{tsx,ts}\"",
     "test:format": "prettier \"**/*.{tsx,ts,md}\" --check",
     "postinstall": "husky install",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "storybook": "start-storybook -p 6006 -s ./",
```

Comment:
simple & beautiful. Thank you for the addition.

Reply:
I'm forwarding this "Thank you" to the great `npx sb init` command 🙏